skimage/feature/corner.py

```diff
@@ -995,7 +995,7 @@ def corner_peaks(image, min_distance=1, threshold_abs=None, threshold_rel=None,
                             threshold_abs=threshold_abs,
                             threshold_rel=threshold_rel,
                             exclude_border=exclude_border,
-                            indices=True, num_peaks=np.inf,
```

Comment:
could you please explain why you had to remove this one?

Reply:
`peak_local_max` has a `remove_arg` decorator stating that this argument will be removed starting in version 0.20. From the docstring to that function:
```
    indices : bool, optional
        If True, the output will be an array representing peak
        coordinates. The coordinates are sorted according to peaks
        values (Larger first). If False, the output will be a boolean
        array shaped as `image.shape` with peaks present at True
        elements. ``indices`` is deprecated and will be removed in
        version 0.20. Default behavior will be to always return peak
        coordinates. You can obtain a mask as shown in the example
        below.
```

Reply:
sorry, was looking at the code in an old branch :-(.